src/fontra/views/editor/editor.js

```diff
@@ -54,7 +55,7 @@ import {
 import { staticGlyphToGLIF } from "../core/glyph-glif.js";
 import { pathToSVG } from "../core/glyph-svg.js";
 import { parseClipboard } from "../core/server-utils.js";
-import Sidebar from "./sidebar.js";
+import Sidebar, { MAX_SIDEBAR_WIDTH, MIN_SIDEBAR_WIDTH } from "./sidebar.js";
```

Comment:
Can we turn `Sidebar` into a "regular" export, instead of a default export? I would love the import line to read like this:

```suggestion
import { Sidebar, MAX_SIDEBAR_WIDTH, MIN_SIDEBAR_WIDTH } from "./sidebar.js";
```